protobuf/scalapb/scalapb.proto

```diff
@@ -284,6 +290,9 @@ message FieldOptions {
   // is not provided. This enables to emulate required fields in proto3.
   optional bool required = 31;
 
+  // If true, complex field would be lazy
```

Comment:
What's the definition of complex? Maybe replace the word complex with its definition?